mini-testing/build.gradle

```diff
@@ -1,11 +1,28 @@
+/*
+ * Copyright 2021 HyperDevs
+ *
+ * Copyright 2020 BQ
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 apply plugin: 'kotlin'
-apply plugin: 'java-library'
 apply from: "../jitpack.gradle"
 
 dependencies {
-    implementation project(":mini-common")
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
+    api project(':mini-common')
+    api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
In `mini-common/build.gradle`  and others `build.gradle` files,this has been changed to `jdk8`,  maybe we should here too?

Reply:
Solved in b6f07929daa04d17efee1757805039f0a85e84b6